pages/checkout-form.html

```diff
@@ -0,0 +1,98 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <link rel="stylesheet" href="../style/style.css">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Lato&family=Tinos:wght@400;700&display=swap" rel="stylesheet">
+    <title>Checkout</title>
+</head>
+<body class="checkout_form">
+<div class="wrapper">
+    <div class="payment_wrapper">
```

Comment:
I see you're using a [BEM](https://ru.bem.info/methodology/)-like approach in class naming, great job 👍 
Usually elements in BEM use double underline `__` prefix, and single underline `_` prefix is used for modificators
```suggestion
    <div class="payment__wrapper">
```